Gem/Settings/ViewModels/ChainNodeViewModel.swift

```diff
@@ -2,23 +2,88 @@
 
 import Foundation
 import Primitives
+import BigInt
+import Components
 
 struct ChainNodeViewModel {
     let chainNode: ChainNode
-    
+
+    private let blockNumber: BigInt?
+    private let latency: LatencyMeasureService.Latency?
+    private let blockNumberError: Error?
+
+    private static let valueFormatter = ValueFormatter.full_US
+
+    init(chainNode: ChainNode, blockNumber: BigInt?, latency: LatencyMeasureService.Latency?, blockNumberError: Error?) {
+        self.chainNode = chainNode
+        self.blockNumber = blockNumber
+        self.latency = latency
+        self.blockNumberError = blockNumberError
+    }
+
     var title: String {
-        guard let host = chainNode.host else {
-            return ""
+        guard let host = chainNode.host else { return "" }
+        return chainNode.isGemNode ? "Gem Wallet Node" : host
+    }
+
+    var titleExtra: String? {
+        let title = Localized.Nodes.ImportNode.latestBlock
+        let emptyTitle = "\(title): -"
```

Comment:
```suggestion
```

let's avoid `emptyTitle` var to simplify formatting. Better to create function `constructTitle(title: String)` or something similar